compiler/scripts/replace.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
Could you give a more specific name to the script? Something like `new_memory_syntax` or `convert_to_new_memory_syntax`? Maybe we'll have other similar scripts in the future.